### DISCUSSION TITLE
How do we document the deployment procedure, especially regarding DB changes ?

### DISCUSSION THREAD

**Discussion by simon-mazenoux** [DIRACGrid/diracx#101] [Developer discussions]
For #64, and unlike DIRAC, diracx require `ON DELETE CASCADE`. My question is, given that this change is specific to diracx, should the migration be documented here or in 8.1 ?

**Comment by fstagni** [comment]
It should be documented here. At the same time if we'll implement https://github.com/DIRACGrid/diracx/issues/88 before the first "installable" DiracX version, then there would be no need for documenting manual actions.

**Comment by marianne013** [comment]
I just had Janusz' try to install diracx. Is there any official documentation yet :-D ?
